test/react_test.rb

```diff
@@ -28,4 +28,23 @@ def test_it_camelizes_props
 
     assert_equal expected_props, React.camelize_props(raw_props)
   end
+
+  def test_it_camelizes_params
```

Comment:
You can actually wrap the test in a defined check like this:

```
if defined?(ActionController::Parameters)
  def test_method
  end
end
```

Which should make it work on Rails <4